plugins/nextcloud/index.php

```diff
@@ -10,7 +10,27 @@ class NextcloudPlugin extends \RainLoop\Plugins\AbstractPlugin
 		DESCRIPTION = 'Integrate with Nextcloud v20+',
 		REQUIRED = '2.36.2';
 
-	public function Init() : void
+	private const IGNORE_SYSTEM_ADDRESSBOOK_KEY = 'ignoreSystemAddressbook';
+	private const IGNORE_SYSTEM_ADDRESSBOOK_DEFAULT_VALUE = true;
+
+	private const ENABLE_NC_ADDRESSBOOK_KEY = 'enableNcAddressbook';
+	private const ENABLE_NC_ADDRESSBOOK_DEFAULT_VALUE = false;
```

Comment:
This doesn't need to be a constant

Think passing false directly is just fine and infact gives faster visibility to someone new to the project about the defaults.. maybe a bit over-refactored here  :p

Reply:
I actually disagree... we are setting it here, so if anybody comes & they make some change, there is a change, by mistake s/he can set wrong default value (here `true` for example).. which can cause regression... using const will block this kind of stuff to happen.